src/elements/content-preview/ContentPreview.js

```diff
@@ -686,6 +690,26 @@ class ContentPreview extends React.PureComponent<Props, State> {
         return !!showAnnotations && (this.canAnnotate() || hasViewAllPermissions || hasViewSelfPermissions);
     }
 
+    /**
+     * Returns whether a preview viewer supports printing
+     *
+     * @return {boolean}
+     */
+    handleCanPrint() {
+        /*
+        Setting this as default so older preview versions can still print documents without having the
+        default state have the print icon.
+         */
+        let canPrint = true;
+        const preview = this.getPreview();
+
+        if (preview && typeof preview.canPrint === 'function') {
```

Comment:
This method is only called in `onPreviewLoad`. Is it possible for preview to be falsy?

Reply:
Not in this scenario because the preview is loaded. But if this gets used outside of that scope it could because this.getPreview will return null if there isn't a preview instance or file, so I felt it better to be safe.